packages/augur-ui/src/modules/markets/selectors/user-markets.ts

```diff
@@ -1,21 +1,39 @@
 import { createSelector } from "reselect";
 import {
   selectLoginAccountAddress,
-  selectMarketTradingHistoryState
+  selectMarketTradingHistoryState,
+  selectPendingQueue,
+  selectCurrentTimestamp
 } from "store/select-state";
+import { CREATE_MARKET } from 'modules/common/constants';
 import selectAllMarkets from "modules/markets/selectors/markets-all";
 import { getLastTradeTimestamp } from "modules/portfolio/helpers/get-last-trade-timestamp";
 import { isSameAddress } from "utils/isSameAddress";
+import { convertUnixToFormattedDate } from "utils/format-date";
 
 export const selectAuthorOwnedMarkets = createSelector(
   selectAllMarkets,
+  selectPendingQueue,
   selectMarketTradingHistoryState,
   selectLoginAccountAddress,
-  (allMarkets, marketTradingHistory, authorId) => {
+  selectCurrentTimestamp,
+  (allMarkets, pendingQueue, marketTradingHistory, authorId, currentTimestamp) => {
     if (!allMarkets || !authorId) return null;
-    const filteredMarkets = allMarkets.filter(
+    let filteredMarkets = allMarkets.filter(
       market => isSameAddress(market.author, authorId)
     );
+    const pendingMarkets = Object.keys(pendingQueue[CREATE_MARKET] || {}).map(key => {
+      let data = pendingQueue[CREATE_MARKET][key];
+      const extraInfo = JSON.parse(data.data._extraInfo);
```

Comment:
I'm not a fan of the `_` in the variable names showing up in a selector. I think those would have been removed when the pendingQueue object was created and saved to state.